app/views/course/object_duplications/new.json.jbuilder

```diff
@@ -28,7 +28,8 @@ json.destinationInstances @destination_instances do |instance|
 end
 
 json.metadata do
-  json.canDuplicateToAnotherInstance can?(:duplicate_to_other_instances, current_tenant)
+  json.canDuplicateToAnotherInstance can?(:duplicate_across_instances, current_tenant) &&
```

Comment:
Separate the logic for `@destination_instances.length > 1`. If you want to disable the view on FE, then check if the destinationInstances is empty there. Dont mix business and auth logic here.

Reply:
I see. okay done for this